homu/main.py

```diff
@@ -1038,9 +1047,9 @@ def create_merge(state, repo_cfg, branch, logger, git_cfg,
                         merge_base_sha))
                     utils.logged_call(git_cmd(
                         '-c',
-                        'user.name=' + git_cfg['name'],
+                        'user.name=' + author_name,
                         '-c',
-                        'user.email=' + git_cfg['email'],
+                        'user.email=' + author_email,
```

Comment:
Don't these need to be escaped?

Reply:
I'm not sure, I'll check. Probably not since we already give a list of arguments and not a single command line.

Reply:
No, it doesn't: arguments are passed as-is. Also note that `squash_msg` isn't escaped either.